### ISSUE TITLE
No audio in Android 10

### ISSUE THREAD

**Issue by jfalexvijay** [BelledonneCommunications/linphone-android#1172]
Hi, I am using linphone 4.3 in my android app. after updating to 4.3 I am getting below issue:

Scenario to reproduce audio issue:
1. App should be running in Android 10.
2. Make call and answer the call from other end.
3. Send the app to background in Android 10. (after 10-30 seconds both the participants will not hear each other)
4. Bring the app to foreground in Android 10. After few seconds, both the participants will hear the audio again.

Not sure what is the issue.

**Comment by Viish** [comment]
Hi,

I think this was caused by the missing foregroundServiceType tag in the Manifest.

I added it to the 4.3.1 release, can you confirm it fixes your issue? 
I tried with a Pixel 3a running Android 11 and I don't reproduce it.

Best regards.

**Comment by jfalexvijay** [comment]
Phone: Samsug Galaxy A21s.
Model: SM-A217F
Android: 10.0
ANDROID SDK: 29
Linphone SDK: 4.5.0-alpha-38-ga907d3001

My app was not receiving incoming calls if the app is running in background for long time (in android 9). I tried removing services and it was working. So I was not using any services in my app. 

It was working fine without any issues. Now I am getting this audio issue only in Android 10. All other versions are fine.

I tried using Service today. But still getting this audio issue.

Plz help me on this.
